docs/src/docs/asciidoc/user/clustered-cache.adoc

```diff
@@ -0,0 +1,68 @@
+---
+---
+= Clustered Cache
+ifndef::sourcedir[]
+include::common.adoc[]
+endif::sourcedir[]
+
+ifdef::notBuildingForSite[]
+include::menu.adoc[]
+endif::notBuildingForSite[]
+
+== Introduction
+
+As with the standalone cache, in order to use the clustered EhCache you will need to configure Clustered cache manager and the cache.
+This is currently in the active development mode and the complete feature to be out very soon, stay tuned!
```

Comment:
Drop this, either it will never appear on the site before the release, or if it does there will be a general indication this is the documentation for the upcoming release.